components/automate-cli/cmd/chef-automate/cmdUtils.go

```diff
@@ -450,7 +465,7 @@ func appendChildFileToParentFile(hostIp, parent, child string) (string, error) {
 	}
 
 	fileContent := string(output)
-	start := fmt.Sprintf("Output for Host IP %s : \n%s\n", hostIp, fileContent)
+	start := fmt.Sprintf("%s\n", fileContent)
```

Comment:
this is needed in other commands, don't remove Host IP

Reply:
Done